emacs.d/b-lsp.el

```diff
@@ -0,0 +1,19 @@
+(provide 'b-lsp)
+
+(require 'package)
+(setq package-user-dir (concat (file-name-directory load-file-name) "melpa"))
+(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)
+(package-initialize)
+
+(require 'lsp-mode)
+(require 'lsp-pyright)
+(setq lsp-completion-provider :none)
+(setq lsp-pyright-auto-import-completions
+      nil)
+(setq lsp-signature-auto-activate nil) ;; disalbe bottom documentation popup
```

Comment:
typo and don't side comment.

Reply:
8715b8e9423034825ab8fe6e8a99d3f9b79c9561